docs/rails-app-deployment.md

```diff
@@ -0,0 +1,72 @@
+# Deploying a Rails App
+
+## Setup GitHub Actions
+1. Open your rails app.
+1. Create the following file as `.github/workflows/deploy-stage.yml`:
+```yaml
+name: Build and Deploy to Stage
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Build Docker image
+    uses: strongmind/public-reusable-workflows/.github/workflows/docker-build.yml@main
+    secrets: inherit
+
+  deploy:
+    name: Deploy Rails to ECS
+    needs: build
+    uses: strongmind/public-reusable-workflows/.github/workflows/rails-deploy.yml@main
+    with:
+      environment-name: stage
+      container-image: ${{ needs.build.outputs.container-image }}
+    secrets: inherit
+```
+3. Create the following file as `.github/workflows/deploy-prod.yml`:
+```yaml
+name: Deploy to production
+
+on: workflow_dispatch
+
+jobs:
+  build:
+    name: Build Docker image
```

Comment:
It feels wrong to build the image again for prod. We need something better right @markng ?